Trisha-tech/OnlineBookSales
# Feature Updated for issue: #638

### Description

Under this, I have added a new glowing card feature to the website book section. I have also added multiple animations and UI improvements inside the website, leading to a better User experience (UX). Also, I have removed an error in the code and fixed the website's navbar, where some of the nav links were duplicated, and the UI was also not that appealing.

- This PR does the following:
  - Adds: New card feature along with animations to the website 
  - Fixes: Errors in the code 
  - Updates: improper UI of the website's nav bar section and also to the body section of the website 

### Related Issues

- Placeholder: "Fixes #638 "

### Changes
List the detailed changes made in this PR.
- Added a new feature to the home section of the website 
- Refactored the design for the navbar 
- Fixed a bug in the code of the entire file which was causing errors on multiple pages and was not allowing the website to run or function properly

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run `npm install` to install dependencies.
3. Run the `npm test` to execute the test suite.
4. Verify that ...

### Screenshots (if applicable)
For navbar:
Before:
![Screenshot 2024-10-31 032629](https://github.com/user-attachments/assets/7c036bed-c546-484b-aa4d-932ace6f3375)
After:
![image](https://github.com/user-attachments/assets/f29ff0a4-15a5-4c2a-8734-68a90b5715de)

For card:

https://github.com/user-attachments/assets/498bbfc0-dce3-480a-b42d-72516d64a510

### Additional Context
Any additional context or information that reviewers should be aware of?
- This PR is based on the following...

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
